consensus/src/tests/bullshark_tests.rs

```diff
@@ -75,20 +75,23 @@ async fn commit_one() {
     let (tx_output, mut rx_output) = channel(1);
     let store_path = test_utils::temp_dir();
     let store = make_consensus_store(&store_path);
+    let gc_depth = 50;
     let bullshark = Bullshark {
         committee: Arc::new(ArcSwap::from_pointee(mock_committee(&keys[..]))),
         store: store.clone(),
-        gc_depth: 50,
+        gc_depth,
     };
     let metrics = Arc::new(ConsensusMetrics::new(&Registry::new()));
     Consensus::spawn(
         Arc::new(ArcSwap::from_pointee(mock_committee(&keys[..]))),
         store,
+        make_certificate_store(&test_utils::temp_dir()),
```

Comment:
Nit: it would be nice, here and in other tests, to follow the pattern of `consensus_store` and declare an intermediate variable for this one above, just to see the stores setup beside each other.